anyway/widgets/widget_utils.py

```diff
@@ -123,3 +125,30 @@ def format_2_level_items(
             series_data.append({BE_CONST.LKEY: l2, BE_CONST.VAL: num})
         res.append({BE_CONST.LKEY: l1, BE_CONST.SERIES: series_data})
     return res
+
+
+def second_level_fill_and_sort(data: dict, default_order: dict) -> dict:

```

Comment:
do we need all 3 methods? or can we merge them into 1?

Reply:
Can be merge to one, what do you think about:

```
    def sort_and_fill_all_levels_of_dict(
        data: defaultdict, num_range: typing.Iterable, default_dict: dict
    ) -> Dict[int, dict]:
        result = {}
        for num in num_range:
            result[num] = default_dict
            level_two_dict = data.get(num, {})
            for key, count in level_two_dict .items():
                result[num][key] += count

        return result 
```